core/src/main/java/tc/oc/pgm/Config.java

```diff
@@ -478,4 +478,39 @@ public static boolean isAppealVisible() {
       return getAppealMessage().length() > 0;
     }
   }
+
+  public static class Fireworks {
+
+    public static class Postmatch {
+      public static boolean enabled() {
+        return getConfiguration().getBoolean("fireworks.post-match.enabled", false);
+      }
+
+      public static int number() {
+        return Math.max(1, getConfiguration().getInt("fireworks.post-match.number", 5));
+      }
+
+      public static int delay() {
+        return Math.max(0, getConfiguration().getInt("fireworks.post-match.delay", 40));
+      }
+
+      public static int frequency() {
+        return Math.max(1, getConfiguration().getInt("fireworks.post-match.frequency", 40));
+      }
+
+      public static int iterations() {
+        return Math.max(1, getConfiguration().getInt("fireworks.post-match.iterations", 15));
+      }
+
+      public static int power() {
+        return Math.max(0, getConfiguration().getInt("fireworks.post-match.power", 2));
+      }
+    }
```

Comment:
It never made any sense to make these configurable, just make these static variables.

Reply:
post-match fireworks are barely used tho. Even pgm 1.9+ never used it except for arcade, ctw/dtc servers never used them